handbook/rbac/rbac.md

```diff
@@ -0,0 +1,128 @@
+
+    When a user runs a JupyterNotebook from a *Report* or dashboard, the Notebook executes with the same privileges as the user. This allows the creation of Notebooks that do not need additional workspace logic. For example the built-in dashboard for Systems Management includes a tile that show the number of alarms for various systems. This tile will only show systems within workspaces for which the current logged in user and is a member. If the user does not have privileges to view alarms, the tile returns no data.
+
+#### Service roles
+
+Service roles apply to SystemLink managed systems and analysis scripts executed by **Analysis Automation**. Systems are always automatically assigned the built-in **Automated Agent** role. Analysis Automation routines can be configured to run with the built-in **Automated Agent** role, a user defined service role, with the privileges of the user initiating the analysis task, or with the TDM user (a Windows user on the SystemLink app server defined in **NI SystemLink TDM Configuration**).
+
+Any custom role may be configured to be a service role at creation by toggling the **Service role** checkbox in the role configuration modal slide out. This cannot be changed after role creation. There are no restrictions on the privileges that may be applied to the role. It exists to signal to various SystemLink UIs what roles can be assigned to users and what roles can be assigned the systems and analysis routines. This forces the user to curtail roles appropriately to each scenario and avoid the proliferation of roles that grant greater access than is necessary to a user, system, or analysis routine.
+
+<figure>
+  <img src="../../img/service-role.png" width="500" />
+  <figcaption>A role configured to be a Service role</figcaption>
+</figure>
+
+### Automatic Data Encapsulation
+
+When a system is added to SystemLink the user must choose which workspace the system will reside. The workspaces available to the user is determine by both their workspace membership and **Add systems** privilege. Once added to a workspace, data produced by the system will automatically be stored in the same workspace as the system. This capability allows users to develop workspace agnostic test applications. Changing workspaces does require changes or redeployment of test application. This is especially helpful when in scenarios such as production verification where the test application cannot change between validation and production but the validation data produced must be kept separate from production data.
+
+## Mapping users to roles in workspaces
+
+Users must have a role within a workspace. To facilitate this, mappings are defined within the context of workspace of the role they are assigned. This allows for users to be assigned different roles in different workspaces if desired.
```

Comment:
```suggestion
Users must be assigned a role within a workspace. ~~~To facilitate this, mappings are defined within the context of workspace of the role they are assigned~~~. This allows for users to be assigned different roles in different workspaces if desired.
```

The sentence surrounded with ~~~ needs to be revisited but I do not have a suggestion.

Reply:
I decided to remove this paragraph, because I could make this same point in the users and roles sections instead.